config/300-pipelinerun.yaml

```diff
@@ -56,6 +56,37 @@ spec:
     # starts to increment
     subresources:
       status: {}
+  - name: v1
+    served: true
```

Comment:
Shouldn't this (and the next line) be `false`?

Reply:
Thanks @abayer.

Reply:
"storage" should also be false